Makefile

```diff
@@ -31,7 +9,7 @@ endif
 
 # Determine whether there's a local yaml installed or use dockerized version.
 # Note in order to install local (faster) yaml: "go get github.com/mikefarah/yaml"
-YAML_CMD:=$(shell which yaml || echo docker run --rm -i $(CALICO_BUILD) yaml)
+YAML_CMD:=$(shell which yaml || echo docker run --rm -i calico/yaml)
```

Comment:
Do we need a `calico/yaml` for each arch?

I guess we support a local yaml, so this is OK.

Reply:
Yes, that was my thinking. It's only used doing a release, and if someone wants to coordinate a release and they're not on `amd64` they should know how to run `go get...`